mava/wrappers/pettingzoo.py

```diff
@@ -556,7 +559,21 @@ def observation_spec(self) -> Dict[str, types.OLT]:
                 observation = _convert_to_spec(
                     self._environment.observation_spaces[agent]
                 )
-                legal_actions = _convert_to_spec(self._environment.action_spaces[agent])
+
+                action_space = self._environment.action_spaces[agent]
+                if type(action_space) == spaces.Discrete:
```

Comment:
To clarify, is this the only action space that the `gym` `_convert_to_spec` methods won't handle properly? All other action spaces are dealt with in the `else` statement.